src/util/eip712.ts

```diff
@@ -210,7 +210,7 @@ export function constructEIP712Tx(doc: CarbonTx.StdSignDoc): EIP712Tx {
     const eip712Tx = {
         types: getTypes(doc.msgs),
         primaryType: "Tx",
-        domain: { ...DEFAULT_CARBON_DOMAIN_FIELDS, chainId: parseChainId(doc.chain_id) },
+        domain: { ...DEFAULT_CARBON_DOMAIN_FIELDS, chainId: parseChainId(chain_id) },
```

Comment:
this should  just be 
```
const { chainId } = await this.syncBlockchain()
domain: { ...DEFAULT_CARBON_DOMAIN_FIELDS, chainId },
```
?

Reply:
edit: okay its util function but u get what i mean

Reply:
yep understood i am passing it as a param right now